### PR TITLE
hide the RoleAndPolicyMenu when anonymous

### PR DESCRIPTION
**Pull request by hobbitronics**
- hide the RoleAndPolicyMenu

### CODE REVIEW

**Review by mtompset — COMMENTED** [review]
Looks good. Though, I can't say I ran it.

File: src/components/AppDrawer.svelte [hunk 1]
```diff
@@ -57,14 +58,16 @@ const logoClickHandler = () => $goto(ROOT)
   title="Covered"
   class="drawer border-white {$showApp ? 'opacity1' : 'opacity0'}"
 >
-  <span class="pointer" on:click={logoClickHandler} slot="header">
+  <span class="pointer" on:click={logoClickHandler} on:keyup slot="header">
```
Comment (Schparky) [inline src/components/AppDrawer.svelte:61]: This change seems unrelated. I also see nothing in the layout that handles the keyup event. Can you help me understand the purpose of this change?

**Review by Schparky — COMMENTED** [review]
I have a question for my own understanding that I'd like answered before I approve the PR.

File: src/components/AppDrawer.svelte (hunk 1 above)
Comment (hobbitronics) [reply src/components/AppDrawer.svelte:61]: non button's that have on:click but no key event cause a warning, but since this doesn't get focus when tabbing it doesn't get used. Perhaps we can use an ignore comment or maybe dig into whether this should be focused by tabbing.

Comment (Schparky) [reply src/components/AppDrawer.svelte]: Either one (ignore or actual fix) would be fine with me. Or even a TODO comment might be fine.

**Review by Schparky — APPROVED** [review]

**Review by forevermatt — APPROVED** [review]

File: src/components/AppDrawer.svelte [hunk 2]
```diff
@@ -57,14 +58,17 @@ const logoClickHandler = () => $goto(ROOT)
   title="Covered"
   class="drawer border-white {$showApp ? 'opacity1' : 'opacity0'}"
 >
+  <!-- svelte-ignore a11y-click-events-have-key-events -->
   <span class="pointer" on:click={logoClickHandler} slot="header">
     <img class="logo" src="/logo.svg" alt="Cover" />
   </span>
```
Comment (forevermatt) [inline src/components/AppDrawer.svelte:64]: I'm okay with proceeding as-is now, but it does bring up a good question of why we're listening for click events on the `span` rather than wrapping the logo image in an `a` link tag. Might be worth revising later.

Comment (hobbitronics) [reply src/components/AppDrawer.svelte:64]: I'll open another PR for that issue. Thanks @forevermatt
